moveit_core/profiler/src/profiler.cpp

```diff
@@ -175,7 +175,7 @@ void Profiler::console()
   std::stringstream ss;
   ss << std::endl;
   status(ss, true);
-  ROS_INFO_STREAM_NAMED("profiler", ss.str());
+  ROS_INFO_STREAM_NAMED("profiler", ss.str().c_str());
```

Comment:
update per this comment:
https://github.com/ros-planning/moveit2/pull/10#discussion_r258226896

Reply:
See above, pushed to a pending issue.